client/packages/workflow-sprotty/css/dark/diagram.css

```diff
@@ -192,4 +192,45 @@ polygon.sprotty-node {
 .sprotty-resize-handle.mouseover {
     stroke: #112;
     stroke-width: 1;
+}
+
+/* Command palette */
```

Comment:
Maybe it would be better to put the command palette related css definitions in a separate file?

Reply:
Good idea, will do! In general, I think it'd be good to extract the sprotty base css and have an own css for the workflow-specific things. We should create a follow up issue for that.